sources/packages/web/src/components/students/applicationTracker/Cancelled.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <application-status-tracker-banner
+    label="You've cancelled your application"
+    icon="fa:fas fa-exclamation-circle"
+    icon-color="danger"
+    background-color="error_bg"
+    ><template #content
+      ><span
+        >You cancelled your application on
+        {{ dateOnlyLongString(applicationDetails?.statusUpdatedOn) }}. However
```

Comment:
We can get rid of this null check by initializing the the ref like this. 
const applicationDetails = ref<CancelledApplicationDetailsAPIOutDTO>({});